### DISCUSSION TITLE
Wrong Android logs location in pinned discussion

### DISCUSSION THREAD

**Discussion by Susko3** [ppy/osu#15366] [Q&A] [answered]
The discussion in question: https://github.com/ppy/osu/discussions/15188

Wrong location: `Android/Data/sh.ppy.osulazer/logs`

Correct location: `Android/data/sh.ppy.osulazer/files/logs`

**Comment by frenzibyte** [comment, marked as answer]
The correct path seems to match [`CONTRIBUTING.md`](https://github.com/ppy/osu/blob/f8aa4c28516cb4126ede24fac7053cf18026a29f/CONTRIBUTING.md). Resolved.
